sdk/servicebus/microsoft-azure-servicebus/cgmanifest.json

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
@conniey If we're generating a manifest as part of the build step should this cgmanifest file be checked in?

Reply:
Yeah, this should be removed. I'll delete it.